### ISSUE TITLE
Upgrade from minecraft_server

### ISSUE THREAD

**Issue by justincm** [MinecraftServerControl/mscs#195]
What is the easiest way to upgrade from minecraft_server to mcsc?

**Comment by sandain** [comment]
Hi @justincm. I'm not sure what you mean about upgrading from minecraft_server. However, if you have a working Minecraft world that you want to import into mscs, take a look at the [installation directions](https://github.com/MinecraftServerControl/mscs#installation) and [word import](https://github.com/MinecraftServerControl/mscs#importing-an-existing-world) sections of the README.md file

**Comment by justincm** [comment]
I am using an older version of your MSCS from 2015, are there any gotcha's in upgrading to the newer version?

**Comment by sandain** [comment]
There may be, but I'll be happy to help you out with any issues that you run into.

The biggest change is that the script has been renamed and the recommended install location has changed from `/home/minecraft/` to `/opt/mscs/`.  We have added support for systemd if you are now using it, but we retained support for the old SysV style startup.  Customization of the script has changed, there should no longer be a need to modify the script, instead we offer the `mscs.defaults` file to change [global properties](https://github.com/MinecraftServerControl/mscs#default-global-server-properties) the `mscs.properties` file to [change settings for individual worlds](https://github.com/MinecraftServerControl/mscs#adjusting-world-properties).

There are some new prerequisites for using the script, so take a look at the [installation instructions](https://github.com/MinecraftServerControl/mscs#installation) before jumping in.  Let me know if you have any issues.
